docs/developer/available-plugins/adyen.mdx

```diff
@@ -0,0 +1,269 @@
+
+Adding a payment form for Adyen requires Saleor to provide available payment methods and client-key. 
+Both of these values can be fetched by calling `checkout.availablePaymentGateways.config`. 
+The query returns all required fields to prepare a payment form with methods available for the checkout.
+
+```graphql
+{
+  checkout(token:"14f11c1c-9bc7-4145-89f7-67b60ef943f2"){
+    availablePaymentGateways{
+      id
+      config{
+        field
+        value
+      }
+    }
+  }
+}
+```
+ As a result, we get details required to render a payment form.
+
+ ```json
+ {
+  "data": {
+    "checkout": {
+      "availablePaymentGateways": [
+        {
+          "id": "mirumee.payments.adyen",
+          "config": [
+            {
+              "field": "client_key",
+              "value": "test_BBDTBL6GWVEJTOXY55TSPJFP54QBIOZ5"
+            },
+            {
+              "field": "config",
+              "value": "{\"groups\": [{\"name\": \"Credit Card\", \"types\": [\"visa\", \"mc\", \"amex\", \"diners\", \"discover\"]}], \"paymentMethods\": [{\"brands\": [\"visa\", \"mc\", \"amex\", \"diners\", \"discover\"], \"details\": [{\"key\": \"encryptedCardNumber\", \"type\": \"cardToken\"}, {\"key\": \"encryptedSecurityCode\", \"type\": \"cardToken\"}, {\"key\": \"encryptedExpiryMonth\", \"type\": \"cardToken\"}, {\"key\": \"encryptedExpiryYear\", \"type\": \"cardToken\"}, {\"key\": \"holderName\", \"optional\": true, \"type\": \"text\"}], \"name\": \"Credit Card\", \"supportsRecurring\": true, \"type\": \"scheme\"}, {\"configuration\": {\"intent\": \"capture\"}, \"name\": \"PayPal\", \"supportsRecurring\": true, \"type\": \"paypal\"}, {\"name\": \"Paysafecard\", \"supportsRecurring\": true, \"type\": \"paysafecard\"}, {\"configuration\": {\"merchantDisplayName\": \"SaleorECOM\", \"merchantIdentifier\": \"1000\"}, \"details\": [{\"key\": \"applepay.token\", \"type\": \"applePayToken\"}], \"name\": \"Apple Pay\", \"supportsRecurring\": true, \"type\": \"applepay\"}, {\"configuration\": {\"merchantId\": \"1000\", \"gatewayMerchantId\": \"SaleorECOM\"}, \"details\": [{\"key\": \"paywithgoogle.token\", \"type\": \"payWithGoogleToken\"}], \"name\": \"Google Pay\", \"supportsRecurring\": true, \"type\": \"paywithgoogle\"}]}"
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+ ```
+
+## Creating payment in Saleor
+
+To create a payment properly, you need to follow the steps described in 
+[Creating a checkout session](developer/checkout.mdx#creating-a-checkout-session) and 
+[Selecting a shipping method](developer/checkout.mdx#selecting-a-shipping-method).
+
+The next step is to choose Adyen as a payment gateway that we want to use to process checkout. 
+For the Adyen payment gateway, we need to provide:
+
+- `checkoutId`:  ID of the checkout, for which new payment should be created.
+
+- `input.gateway`:  ID of the payment gateway which should be assigned to this payment. For Adyen it will be:  `mirumee.payments.adyen`.
```

Comment:
```suggestion
- `input.gateway`: ID of the payment gateway which should be assigned to this payment. For Adyen it will be:  `mirumee.payments.adyen`.
```